ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/MergeTransitionConfigCheck.java

```diff
@@ -106,16 +106,14 @@ private synchronized void verifyTransitionConfiguration() {
                       .equals(remoteTransitionConfiguration.getTerminalTotalDifficulty())
                   || !localTransitionConfiguration
                       .getTerminalBlockHash()
-                      .equals(remoteTransitionConfiguration.getTerminalBlockHash())) {
+                      .equals(remoteTransitionConfiguration.getTerminalBlockHash())
+                  || !localTransitionConfiguration
+                      .getTerminalBlockNumber()
```

Comment:
I have doubts here. As far as I remember, the local transition configuration doesn't know anything about block number. There is no block number configuration on the CL side. The block number is only configured on EL side so it only appears only in `remoteTransitionConfiguration`

Reply:
OMG I forgot to push it. Fixed